pkg/comp-functions/functions/vshnpostgres/scripts/queries.yml

```diff
@@ -0,0 +1,880 @@
+pg_replication:
```

Comment:
Where is this coming from? How can we ensure this is updated in time?

Reply:
This is from the original stackgres exporter config.

I did a `curl localhost:9187/metrics | grep -o ".*{" | uniq -c | sort -bgr` on the container to identify all the really big metrics and just removed them from the configs.

As for updating them... We could probably observe the original confing in the compfunction and remove the broken queries then dynamically 🤔